codecov.yaml

```diff
@@ -15,4 +15,4 @@ parsers:
   javascript:
     enable_partials: yes
 ignore:
-  - 'coverage/*.json'
+  - '**/*.json'
```

Comment:
this is the only thing the PR mentions. I'd expect it at least to be in its own commit, and the rest in a `ci: refactor`.